### ISSUE TITLE
New dataset, accuracy is very low  0.2- 0.3

### ISSUE THREAD

**Issue by anathema43** [Kazuhito00/hand-gesture-recognition-using-mediapipe#5]
Hi, 
Taking the footprint of this work , i created my won getsure dataset from media pipe, did the same preprocessing as was written and trained a 3 dense layer classifier  with adam and categorical cross entropy , but my accuracy doesnt increase beyond 0.3,  I have more than 1k dataset for each class.  i am baffled as to why is it not working? any input from your side would be useful

Preprocessing step-    #shape(N,21,2)
def preprocess(data):
    for i in range(data.shape[0]):
        data[i,:,0]=data[i,:,0] - data[i,0,0]
        data[i,:,1] =data[i,:,1]- data[i,0,1]
        data[i]= data[i]/np.amax(np.abs(data[i]))
    data=data.reshape(data.shape[0],42)
    return data 
# model
def get_model():
    reg= regularizers.l1(0)
    inp= Input(shape=(42,))
    x= Dense(32,activation='relu',activity_regularizer=reg, kernel_initializer='he_uniform')(inp)
    x= Dropout(0)(x)
    x= Dense(16,activation='relu',activity_regularizer=reg, kernel_initializer='he_uniform') (x)
    x= Dropout(0)(x)
    x= Dense(8,activation='relu',activity_regularizer=reg, kernel_initializer='he_uniform')(x)
    out= Dense(5,activation='softmax')(x)
    model= Model(inp,out)
    opt = tf.keras.optimizers.Adam(learning_rate=0.1)
    model.compile(optimizer=opt,loss='categorical_crossentropy',metrics=['accuracy'])
    model.summary()
    return model

**Comment by Kazuhito00** [comment]
The code you wrote doesn't seem to be what this repository provides.
Also, I can't give you any advice because I don't know the contents of the dataset.

**Comment by Kazuhito00** [comment]
It will close because time has passed.
If you have another problem, please create the issue again.
